NewRoutes/bookings.js

```diff
@@ -0,0 +1,115 @@
+//routes
+const express = require('express');
+const router = express.Router();
+const Bookings = require('../models/Bookings')  
+
+
+//get all Bookings 
+router.get('/', async (req, res) => {
+    try{
+        const booking = await Bookings.find()
+        res.json(booking)
+    }
+    catch
+        (err){
+            res.status(500).json({message:err.message})
+        } 
+ });
+  
+
+ //get one booking
+router.get('/:id', getBooking,(req,res)=> {
+    res.json(res.booking)
+})
+
+
+ ///updating one booking
+
+ router.patch('/:id', getBooking, async (req, res) => {
+    if(req.body.clientName!=null){
+        res.booking.clientName = req.body.clientName
```

Comment:
read about object destructuring 
https://javascript.info/destructuring-assignment#object-destructuring
instead of using `res.booking...` you can do `const { date, bookingNum, restaurantNum  }  = res.booking`